bin/desi_proc

```diff
@@ -271,6 +279,7 @@ if args.batch:
         fx.write('fi\n')
     
     print('Wrote {}'.format(scriptfile))
+    print('logfile will be {}/{}-%j.log\n'.format(batchdir, jobname))
```

Comment:
minor: the %j in the slurm batch file gets replaced with the JOBID at runtime.  We don't know that jobid yet, but this printout would be clearer with "-JOBID" instead of "-%j".